lib/qa/authorities/linked_data/find_term.rb

```diff
@@ -75,13 +79,18 @@ def expects_uri?
           term_config.term_id_expects_uri?
         end
 
+        def extract_uri_from_id(id)
+          return @uri = id if expects_uri?
```

Comment:
For authorities that send ID to findterm instead of an URI, this finds the URI given the ID.  Uses the new method in the graph service to find the appropriate triple.